_data/bib_stats_msc.yml

```diff
@@ -0,0 +1,294 @@
+# Autogenerated by GAP
```

Comment:
It wasn't really generated by GAP, but rather by a script. Having the name of that script here would be very helpful for anybody who wants to figure out how this all works.

Reply:
Changed to `Autogenerated by Doc/Bib/convbib.g`